ui-core/src/styles/inputs/slider.css

```diff
@@ -0,0 +1,210 @@
+      box-shadow: 0px 1px 3px theme('colors.black.10');
+      background-color: theme('colors.white.100');
+      opacity: 1;
+    }
+
+    &:focus {
+      outline: none;
+
+      &::-webkit-slider-runnable-track {
+        box-sizing: content-box;
+        border-radius: 4px;
+        background-color: theme('colors.white.75');
+        border: 2px solid theme('colors.primary.50');
+        box-shadow: 0px 0px 0px 0.5px theme('colors.black.25') inset;
+        opacity: 1;
+      }
+
+      &::-moz-range-track {
+        box-sizing: content-box;
+        border-radius: 4px;
+        background-color: theme('colors.white.75');
+        border: 2px solid theme('colors.primary.50');
+        box-shadow: 0px 0px 0px 0.5px theme('colors.black.25') inset;
+        opacity: 1;
+      }
+
+      &::-webkit-slider-thumb {
+        -webkit-appearance: none;
+        appearance: none;
+        width: 10px;
+        height: 10px;
+        margin-top: -3px;
+        border-radius: 50%;
+        background-color: theme('colors.white.100');
+        border: 1px solid theme('colors.grey.90');
+        box-shadow: 0px 1px 3px theme('colors.black.10');
+        opacity: 1;
+      }
+      
+      &::-moz-range-thumb {
+        -webkit-appearance: none;
+        appearance: none;
+        width: 10px;
+        height: 10px;
+        margin-top: -3px;
+        border-radius: 50%;
+        background-color: theme('colors.white.100');
+        border: 1px solid theme('colors.grey.90');
+        box-shadow: 0px 1px 3px theme('colors.black.10');
+        opacity: 1;
+      }
+    }
+
+    &:active {
+      outline: none;
+
+      &::-webkit-slider-runnable-track {
+        box-sizing: content-box;
+        border: 0;
+        background-color: theme('colors.white.100');
+        border-radius: 2px;
+        box-shadow: 0px 0px 0px 0.5px theme('colors.black.75') inset;
+        opacity: 1;
+      }
+
+      &::-moz-range-track {
+        box-sizing: content-box;
+        border: 0;
+        background-color: theme('colors.white.100');
+        border-radius: 2px;
+        box-shadow: 0px 0px 0px 0.5px theme('colors.black.75') inset;
+        opacity: 1;
+
+      }
+
+      &::-webkit-slider-thumb {
+        -webkit-appearance: none;
+        appearance: none;
+        width: 8px;
+        height: 8px;
+        margin-top: -3px;
+        border-radius: 50%;
+        background-color: theme('colors.white.100');
+        border: 1px solid theme('colors.primary.70');
+        box-shadow: 0px 0.5px 1px theme('colors.black.25');
+        opacity: 1;
+      }
+      
+      &::-moz-range-thumb {
+        -webkit-appearance: none;
+        appearance: none;
+        width: 10px;
+        height: 10px;
+        margin-top: -3px;
+        border-radius: 50%;
+        background-color: theme('colors.white.100');
+        border: 1px solid theme('colors.primary.70');
+        box-shadow: 0px 0.5px 1px theme('colors.black.25');
+        opacity: 1;
+      }
+    }
+
+    &:hover {
```

Comment:
We should not have the hover behavior when the parent (`.range-wrapper`) has `disabled` class. Maybe we can do something with `:not()`